spec/lib/metadata_listener/fits_service_spec.rb

```diff
@@ -37,13 +37,13 @@
     end
   end
 
-  context 'with errors during the HTTP request' do
-    before { ENV['FITS_ENDPOINT'] = 'http://localhost:8080/fits/wrong' }
+  context 'with errors during the FITs call' do
+    before { ENV['FITS_PATH'] = '/usr/share/fits/fits.broken' }
 
-    after { ENV['FITS_ENDPOINT'] = 'http://localhost:8080/fits/examine' }
+    after { ENV['FITS_PATH'] = '/usr/share/fits/fits.sh' }
 
     it 'raises an error' do
-      expect { service }.to raise_error(StandardError, 'HTTP Status 404 – Not Found')
+      expect { service }.to raise_error(StandardError)
```

Comment:
I think this should be `FitsError`. It's probably passing because FitsError is a subclass of StandardError. 🤷‍♂️

Reply:
I think so, that's how it originally was, so I left it.

Reply:
Because we're raising a FitsError if the Open3 call to Fits is not successful, the test sould be:

``` ruby
expect { service }. to raise_error(FitsError, 'whatever stderr is here')
```
It'll probably look something like "command not found: fits.broken"